lib/LoginSchema.ts

```diff
@@ -0,0 +1,11 @@
+import { z } from "zod";
+
+export type LoginSchema = z.infer<typeof LoginSchema>;
+
+export const LoginSchema = z.object({
```

Comment:
카멜케이스로 표현해주는게 좋을 것 같네요~
```suggestion
export const loginSchema = z.object({
```